tests/core/transport/test_websocket.py

```diff
@@ -7,10 +7,10 @@
 import pytest
 
 if hasattr(builtins, "ExceptionGroup"):
-    ExceptionGroup = builtins.ExceptionGroup
+    ExceptionGroup = builtins.ExceptionGroup  # type: ignore[misc]
 else:
     # Fallback for older Python versions
-    ExceptionGroup = Exception
+    ExceptionGroup = Exception  # type: ignore[misc]
```

Comment:
Removing these type ignores doesn't cause any error for me locally. Are they still necessary?